### PR TITLE
feat(saas): document the upcoming London region

### PR DESCRIPTION
**Pull request by multani**
## Description

The "dual-region backup" feature will not be available in this region.

See: https://github.com/camunda/product-hub/issues/2315

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineerring DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by multani** [comment]
Hello :wave:

> These files were changed only in _docs/_. You might want to duplicate these changes in _versioned_docs/version-8.5/_.
>     * docs/reference/regions.md

I did the change only in `docs/`, but this new region should be available in the coming weeks ; I'm not sure if I should also update the 8.5 release doc :thinking:

**Comment by akeller** [comment]
> I did the change only in `docs/`, but this new region should be available in the coming weeks ; I'm not sure if I should also update the 8.5 release doc 🤔

@multani since this is a SaaS feature, I imagine it could be available for any supported version. Will this new region only support 8.6+ clusters?

File: docs/reference/regions.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Below, find a list of regions currently supported in Camunda 8 SaaS.
 | ------------------------------------------------ | ------------------------------------------------- |
 | Belgium, Europe (europe-west1)                   | Germany, Europe (europe-west3)                    |
 | Iowa, North America (us-central1)                | Salt Lake City, North America (us-west1)          |
+| London, Europe (europe-west2)                    | *Not available*                                   |
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/reference/regions.md]: The way you have documented this is fine -- so long as we are consistent, the formatting doesn't matter too much.

**Review by than-pet — DISMISSED** [review]

**Comment by akeller** [comment]
Based on @than-pet's comment in the epic, it sounds like this was released... today? Do we need to get this PR backported to all supported versions and released today as well?

**Comment by than-pet** [comment]
> Based on @than-pet's comment in the epic, it sounds like this was released... today? Do we need to get this PR backported to all supported versions and released today as well?

Yes, we have to backport it, but there is no urgency to merge this PR today

**Comment by christinaausley** [comment]
@multani @than-pet Would you like to see this merged before release tomorrow? Is this ready for final review, and can we move it out of `draft`?

**Review by than-pet — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**